src/atlas/interpolation/method/fe/FiniteElement.h

```diff
@@ -28,7 +28,9 @@ namespace method {
 
 class FiniteElement : public Method {
 public:
-    FiniteElement(const Config& config): Method(config) {}
+    FiniteElement(const Config& config): Method(config) {
+        if (config.has("max_fraction_elems_to_try") ) { config.get("max_fraction_elems_to_try", maxFractionElemsToTry_); }
```

Comment:
You can just use
```
config.get("max_fraction_elems_to_try", maxFractionElemsToTry_);
```
The default value is already initialised automatically with line 84, and 'config.get(key,value)' will only change the value if the key is present.

Reply:
I see, I missed that. I have made the suggested change in the latest commit.